### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by bgrozev**
Uses consistent naming (Participant -> participant, WebDriver -> driver), fixes formatting, simplifies code and exports constants, moves some code from utils to Participant.

### CODE REVIEW

**Comment by bgrozev** [comment]
Jenkins test again please

**Comment by damencho** [comment]
Automatic merge failed; fix conflicts and then commit the result.
Apparently, there is a branch refactor in jvb which has a merge conflict.

**Comment by bgrozev** [comment]
Yeah, I closed the conflicting PR in jitsi-videobridge.

File: src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java [hunk 1]
```diff
@@ -94,37 +97,40 @@ private void assertOneDominantSpeaker(WebDriver driver)
...
 
     private void muteAllParticipants()
     {
         MuteTest muteTest = new MuteTest(this);
-        muteTest.muteOwnerAndCheck();
-        muteTest.muteParticipantAndCheck();
-        muteTest.muteThirdParticipantAndCheck();
+        muteTest.muteParticipant1AndCheck();
+        muteTest.muteParticipant2AndCheck();
+        muteTest.muteParticipant3AndCheck();
```
Comment (bbaldino) [inline src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java]: 🥇

Comment (bgrozev) [reply src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java]: I normalized the names to "participant1", "participant2" and "participant3". For example, in different places we used to refer to participant2 as: "participant", "secondParticipant", "peer", "peer2" and "2nd".

**Comment by paweldomas** [comment]
Thanks for doing that ! I was thinking about that, but it's so many places to change...

**Pull request merged**